tests/settings/utils_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+
+from __future__ import absolute_import
+
+import unittest
+
+import mock
+
+from alot.settings import utils
+
+
+class TestResolveAtt(unittest.TestCase):
+
+    __patchers = []
+    fallback = mock.Mock()
+    fallback.foreground = 'some fallback foreground value'
+    fallback.background = 'some fallback background value'
+
+    @classmethod
+    def setUpClass(cls):
+        cls.__patchers.append(mock.patch('urwid.AttrSpec', mock.Mock(
```

Comment:
so, when you do an import, you get a shallow copy (a group of pointers basically) back to the original module. That way if someone does something stupid in their packge like:
```python
import os

os.path.join = lambda *a: '/'.join('foo', *a)
```

then that doesn't propegate out of their module. So even if that module is imported into your module and then you import `os`, you'll get a new shallow copy of the canoncial `os` module.